packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -401,6 +401,23 @@ def check_diagnostics_metrics():
         check_diagnostics_metrics()
 
 
+def test_metrics_fluentbit(dcos_api_session):
+    """Ensure that fluent bit metrics are present on masters and agents"""
+    nodes = get_master_and_agents(dcos_api_session)
+
+    for node in nodes:
+        @retrying.retry(wait_fixed=STD_INTERVAL, stop_max_delay=METRICS_WAITTIME)
+        def check_fluentbit_metrics():
+            response = get_metrics_prom(dcos_api_session, node)
+            for family in text_string_to_metric_families(response.text):
+                for sample in family.samples:
+                    if sample[0].startswith('fluentbit_output_errors_total'):
+                        assert sample[1]['dcos_component_name'] == 'DC/OS Fluent Bit'
+                        return
```

Comment:
We should remove this `return` statement since test will pass even if fluentbit is working only on a single node.